src/components/theme/PreviewImage/PreviewImage.jsx

```diff
@@ -7,13 +7,23 @@ import DefaultImageSVG from '@plone/volto/components/manage/Blocks/Listing/defau
 /**
  * Renders a preview image for a catalog brain result item.
  */
-function PreviewImage({ item, alt, ...rest }) {
+function PreviewImage({
+  item,
+  alt,
+  image_field,
+  show_default = true,
```

Comment:
We use camel case for names defined in javascript, so this should be showDefault. (`image_field` uses an underscore because it's the name of a field from the backend)